cmd/internal/cli/pull.go

```diff
@@ -293,6 +293,7 @@ func pullRun(cmd *cobra.Command, args []string) {
 					// exit status 10 after replying no
 					exitStat = 10
 				}
+				fmt.Fprintf(os.Stderr, "You can avoid this by using the '-U' flag\n")
```

Comment:
what is "this"? Does it refer to the message above on line 282? If yes, can the message be moved there?